doc/reference/modules/manipulating_data.xml

```diff
@@ -680,9 +683,9 @@ secondSession.SaveOrUpdate(mate);  // save the new instance (mate has a null id)
                 <listitem>
                     <para>
                         if the object is versioned (<literal>version</literal> or
-                        <literal>timestamp</literal>), then the version will take precedence
-                        to identifier check, unless the versions
-                        <literal>unsaved-value="undefined"</literal> (default value)
```

Comment:
No more the case since [NH-3505](https://nhibernate.jira.com/browse/NH-3505), see #1354.